### PR TITLE
Simplify

### PR DESCRIPTION
**Pull request by Christian-Oleson**

### CODE REVIEW

File: generator.js [hunk 1]
```diff
@@ -11,23 +11,24 @@ const moodTwo = 'happy'
 const moodThree = 'sleepy'
 const moodFour = 'sad'
```
Comment (Christian-Oleson) [inline generator.js:12]: At this point, we are declaring constant strings just to embed them in an array of strings. We'd just be better off hard coding them there.

File: generator.js [hunk 2]
```diff
@@ -11,23 +11,24 @@ const moodTwo = 'happy'
...
 
-const bodyArr = [bodyOne,bodyTwo,bodyThree];
-const colorArr = [colorOne,colorTwo,colorThree,colorFour,colorFive];
+const bodyArr = [bodyOne, bodyTwo, bodyThree];
+const colorArr = [colorOne, colorTwo, colorThree, colorFour, colorFive];
```
Comment (Christian-Oleson) [inline generator.js:15]: spacing in an array just makes it more readable

File: generator.js [hunk 3]
```diff
@@ -11,23 +11,24 @@ const moodTwo = 'happy'
...
-};
+function arrayNumberRandomizer(array) {
+    return Math.floor((Math.random() * array.length));
+}
 
-generator(bodyArr,colorArr,moodArr);
\ No newline at end of file
+function arrayRandomizer(array) {
+    return array[arrayNumberRandomizer(array)]
+}
```
Comment (Christian-Oleson) [inline generator.js:32]: This way, if your array randomizer logic changes, you change it once, not 3 times

File: generator.js [hunk 4]
```diff
@@ -11,23 +11,24 @@ const moodTwo = 'happy'
...
-    let newMood = moodArr[ran3]
+    console.log(`Your monster is ${newBody}, ${newColor}, and ${newMood}!`);
+}
 
-    console.log(
-        `Your monster is ${newBody}, ${newColor}, and ${newMood}!`
-    );
-};
+function arrayNumberRandomizer(array) {
+    return Math.floor((Math.random() * array.length));
```
Comment (Christian-Oleson) [inline generator.js:27]: This way, if your array length logic changes, you change it once, not 3 times

File: .gitignore [hunk 5]
```diff
@@ -0,0 +1,2 @@
+.vs/
+.idea/*
```
Comment (Christian-Oleson) [inline .gitignore:2]: I use webstorm, so I'm ignoring those files from git so they aren't added to the repo

File: generator.js [hunk 6]
```diff
@@ -1,4 +1,4 @@
-const bodyOne = "bald";
```
Comment (Christian-Oleson) [inline generator.js:1]: unneeded ;

**Comment by philip-mcwhorter** [comment]
Where do you go in GitHub to see the updated code before committing? I just see the comments.

**Comment by Christian-Oleson** [comment]
> Where do you go in GitHub to see the updated code before committing? I just see the comments.

Go to the files tab

**Pull request merged**